test/validate/windows_validate.go

```diff
@@ -89,6 +98,17 @@ func hnsStateFileIps(result []byte) (map[string]string, error) {
 	return hnsPodIps, nil
 }
 
+// return windows HNS network state
+func hnsNetworkState(result []byte) ([]HNSNetwork, error) {
+	var hnsNetworkResult []HNSNetwork
+	err := json.Unmarshal(result, &hnsNetworkResult)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to unmarshal hns network list")
```

Comment:
```suggestion
		return nil, errors.Wrap(err, "failed to unmarshal hns network list")
```
since you are not using any formatting directives. or otherwise
```suggestion
		return nil, errors.Wrapf(err, "failed to unmarshal hns network list. hns bytes: %s", string(result))
```